.rubocop.yml

```diff
@@ -0,0 +1,33 @@
+# require: rubocop-rspec
```

Comment:
I love that I didn't have my ruby project to extend rubocop set up to actually use rubocop 😵